sources/packages/web/src/components/generic/FormioModalDialog.vue

```diff
@@ -8,7 +8,9 @@
       ></formio>
     </template>
     <template v-slot:footer>
-      <slot name="actions" :cancel="cancel" :submit="submit"></slot>
+      <div class="mr-3">
```

Comment:
I assume this margin space is to add for the button which is inside the slot to be aligned. IMO the margin when added to a horizontal container can be added for both left and right. 

e.g                   [                  ]
witth margin       [           ]

Reply:
I will change it to be consistent.

Reply:
Changed to `mx-3`.